tests/test_service_factory.py

```diff
@@ -27,7 +46,7 @@ def test_service_factory(self):
 
         # Test services config
         services = ServiceFactory().get_active_services()
-        assert services[0].name == "Magpie"
-        assert services[1].name == "Geoserver"
-        assert services[2].name == "Thredds"
-        assert services[3].name == "Nginx"
+        assert services[0].name == "Geoserver"
+        assert services[1].name == "Magpie"
+        assert services[2].name == "Nginx"
+        assert services[3].name == "Thredds"
```

Comment:
similar to my other comment
order seems somewhat random and prone to break the test

Reply:
Here at least the config is set in the setUpClass method in the test itself

Reply:
But the reported order here is not even the same as the one in that setUpClass config.
I'm not even sure how this order is obtained other that hash happen to be this way this time, or the dict keys are rewritten somewhere since python 3 is supposed to preserve insert order.